### ISSUE TITLE
Create a derivative user collection in `hasadna` to that in `startup-os`

### ISSUE THREAD

**Issue by oferb** [google/startup-os#563]
`startup-os` has a user collection:
https://github.com/google/startup-os/blob/master/reviewer_config.prototxt

We'd like `hasadna` to have a derivative one. It means that a user can be:
* Defined only in `startup-os`
* Defined only in `hasadna`
* Defined in both, in which case we merge the data. We give precedence to `startup-os`: for fields like name and ID we will use `startup-os`, for fields like contributions we will merge the lists.

The code to merge the data and upload it to Firestore is here:
https://github.com/google/startup-os/blob/master/tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

So basically, this task is to update the code there to reflect the above.
